### ISSUE TITLE
Saved Views (Feature request) 

### ISSUE THREAD

**Issue by nosbig** [zadam/trilium#4312]
### Describe feature

I would love to be able to save the view state of the displayed notes for later recall as I am intermittently working on new projects.  In my use case, I have three separate notes for reference in a single tree.  I have the sidebar collapsed (although that's not a major point of concern.  I have two splits, so three panes, each with a different specific note related to the work which I am doing.  I am sure that others would prefer to have a tab setup be able to be saved as well as splits and note selections.

I am using v0.60.4 for the time being on Debian Linux (via the .deb package) with the same version sync server to share the notes.

For now, if I need something other than the setup I have established, I log into the web interface of the sync server, so I have a workaround.

That would require surfacing some sort of UI for switching views, reverting to "normal" (maybe "normal" is just a default view), and deleting unneeded views when a given project or task might be finished.

**Comment by zadam** [comment]
Hi, why not just keep the tab with the 2 splits (3 panes) open forever? If you need something else, do it in another tab?

**Comment by nosbig** [comment]
Well, that's interesting...  I didn't realize that tabs were an organizational level above splits.  That will totally work for my use case.  Maybe others might find saved views/workspaces useful in the future, but I have an excellent way to tackle my needs.

Thanks for an awesome project!

**Comment by zadam** [comment]
Yeah, some apps have tabs within splits, but trilium has splits within tabs ...

**Comment by eliandoran** [comment]
@zadam 's recommendation should suffice.
